core/examples/luigi-example-js/package.json

```
@@ -4,18 +4,18 @@
   "description": "",
   "main": "public/index.html",
   "scripts": {
-    "build": "webpack --entry ./luigi-config.js -o ./public/luigi-config.js --mode production",
+    "build": "webpack --entry ./luigi-config.js --mode production",
     "start": "live-server --entry-file=index.html public",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "license": "ISC",
   "dependencies": {
-    "@luigi-project/client": "^1.22.0",
-    "@luigi-project/core": "^1.22.0",
-    "copy-webpack-plugin": "^5.1.1",
-    "@sap-theming/theming-base-content": "^11.1.38",
-    "fundamental-styles": "^0.23.0",
-    "webpack": "^4.41.5",
-    "webpack-cli": "^3.3.10"
+    "@luigi-project/client": "^1.25.1",
+    "@luigi-project/core": "^1.25.1",
+    "@sap-theming/theming-base-content": "^11.1.44",
+    "copy-webpack-plugin": "^11.0.0",
+    "fundamental-styles": "^0.23.1",
+    "webpack": "^5.75.0",
+    "webpack-cli": "^5.0.0"
```

If you don't have installed the live-server globally you can't start the example. We should add it to the dependencies.
In addition we could also use _devDependencies_ and sort the dependencies.
```suggestion
    "webpack-cli": "^5.0.0",
    "live-server": "1.2.2"
```